### PR TITLE
Updated command instructions with examples for rad_install_kubernetes.md page

### LINKED ISSUE #7968: Add more detail to rad install kubernetes doc
## What content needs to be created or modified?
The [`rad install kubernetes`](https://docs.radapp.io/reference/cli/rad_install_kubernetes/) document explains that the command has the `--set` and `--set-file` options. We need more information in the text that explains what the available values are or how to discover the available values that can be set.

## Describe the solution you'd like
Provide an explanation of the available values or a link to another API command that allows a user to discover the values. Provide an example as well.

## Where should the new material be placed?
Include in the existing help document.

## The associated pull request from the Radius code repo, if applicable
N/A

## Additional context
N/A

[AB#13253](https://dev.azure.com/azure-octo/e61041b4-555f-47ae-95b2-4f8ab480ea57/_workitems/edit/13253)

### PR DESCRIPTION
**Pull request by ckansara16**
# Description

Enhanced examples with different arguments for 'rad install kubernetes' command requested in https://github.com/radius-project/radius/issues/7968 

## Type of change

- This pull request is a minor refactor, code cleanup, test improvement, or other maintenance task and doesn't change the functionality of Radius (issue link optional).

Fixes: #7968 

## Contributor checklist
Please verify that the PR meets the following requirements, where applicable:

- [ ] An overview of proposed schema changes is included in a linked GitHub issue.
- [ ] A design document PR is created in the [design-notes repository](https://github.com/radius-project/design-notes/), if new APIs are being introduced.
- [ ] If applicable, design document has been reviewed and approved by Radius maintainers/approvers.
- [ ] A PR for the [samples repository](https://github.com/radius-project/samples) is created, if existing samples are affected by the changes in this PR.
- [X] A PR for the [documentation repository](https://github.com/radius-project/docs) is created, if the changes in this PR affect the documentation or any user facing updates are made.
- [ ] A PR for the [recipes repository](https://github.com/radius-project/recipes) is created, if existing recipes are affected by the changes in this PR.

### CODE REVIEW

File: pkg/cli/cmd/install/kubernetes/kubernetes.go [hunk 1]
```diff
@@ -56,6 +56,18 @@ rad install kubernetes --set key=value
...
+rad install kubernetes --set global.zipkin.url=http://localhost:9411/api/v2/spans
+
+# Install Radius with central prometheus monitoring service
+rad install kubernetes --set global.prometheus.path=/customdomain.com/metrics,global.prometheus.port=443,global.rootCA.cert=/path/to/rootCA.crt 
+
+# Install Radius from helmchart referred from path
+rad install kubernetes --chart /root/radius/deploy/Chart
+
+# Re-install Radius with latest version
+rad install kubernetes --reinstall
```
Comment (rynowak) [inline pkg/cli/cmd/install/kubernetes/kubernetes.go:70]: /cc @Reshrahim

File: pkg/cli/cmd/install/kubernetes/kubernetes.go [hunk 2]
```diff
@@ -56,6 +56,18 @@ rad install kubernetes --set key=value
...
+# Install Radius with zipkin server for distributed tracing 
+rad install kubernetes --set global.zipkin.url=http://localhost:9411/api/v2/spans
+
+# Install Radius with central prometheus monitoring service
+rad install kubernetes --set global.prometheus.path=/customdomain.com/metrics,global.prometheus.port=443,global.rootCA.cert=/path/to/rootCA.crt 
+
+# Install Radius from helmchart referred from path
+rad install kubernetes --chart /root/radius/deploy/Chart
+
+# Re-install Radius with latest version
```
Comment (Reshrahim) [inline pkg/cli/cmd/install/kubernetes/kubernetes.go:69]:
```suggestion
# Force re-install Radius with latest version
```

**Review by Reshrahim — COMMENTED** [review]
Thanks for the contribution. Suggested some minor tweaks

**Comment by ckansara16** [comment]
> Thanks for the contribution. Suggested some minor tweaks

@Reshrahim updated.

**Review by Reshrahim — APPROVED** [review]

**Review by lakshmimsft — APPROVED** [review]

**Pull request merged**